### PR TITLE
FTB Modpack Downloading Blog/News Post

### PR DESCRIPTION
**Pull request by IThundxr**

### CODE REVIEW

File: src/news/ftb-removal.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+### Initial Permission Request
+
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request1.png" style="width: 40em">
+<br/>
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request2.png" style="width: 40em">
+<br/>
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request3.png" style="width: 40em">
+
+### Request to Remove FTB  Modpack Downloading
```
Comment (Scrumplex) [inline src/news/ftb-removal.md:31]: two spaces

File: src/news/ftb-removal.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request6.png" style="width: 40em">
+<br/>
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request7.png" style="width: 40em">
+<br/>
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request8.png" style="width: 40em">
+
+~ PrismLauncher Team
+
+
```
Comment (Scrumplex) [inline src/news/ftb-removal.md:47]: too many newlines. Run markdownlint

File: src/news/ftb-removal.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+[Github-PR]: https://github.com/PrismLauncher/PrismLauncher/pull/1040
```
Comment (Scrumplex) [inline src/news/ftb-removal.md:48]: nit: rename to `pr-ftb-removal` or smth like that

**Review by Scrumplex — CHANGES_REQUESTED** [review]

**Review by txtsd — CHANGES_REQUESTED** [review]
Insert a newline at end of file please

File: src/news/ftb-removal.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+---
+title: FTB Modpack Downloading Removal.
+description: Why FTB Modpack Downloading was Removed.
+date: 2023-04-27
+tags:
+  - News
+---
+
+Due to a request from Slowpoke on behalf of the FTB team, we've had to remove the FTB Modpack downloading feature from PrismLauncher. [Github PR][pr-ftb-removal]
```
Comment (DioEgizio) [inline src/news/ftb-removal.md:9]:
```suggestion
Due to a request from Slowpoke on behalf of the FTB team, we've had to remove the FTB Modpack downloading feature from Prism Launcher. [Github PR][pr-ftb-removal]
```

File: src/news/ftb-removal.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+At the request of Slowpoke, representing the FTB team, we have removed the feature that allows downloading FTB modpacks from Prism Launcher. Github PR
```
Comment (DioEgizio) [inline src/news/ftb-removal.md:11]: Huh?

Comment (DioEgizio) [reply src/news/ftb-removal.md:11]: Just remove this line

File: src/news/ftb-removal.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+title: FTB Modpack Downloading Removal.
+description: Why FTB Modpack Downloading was Removed.
+date: 2023-04-27
+tags:
+  - News
+---
+
+At the request of Slowpoke, representing the FTB team, we have removed the feature that allows downloading FTB modpacks from Prism Launcher. [Github PR][pr-ftb-removal]
+
+Existing instances of FTB modpacks will remain functional, but once the PR is merged, downloading FTB packs directly will no longer be possible.
```
Comment (txtsd) [inline src/news/ftb-removal.md:11]: Muggles don't know what merging a PR is. Say "with the next release" instead.

Comment (IThundxr) [reply src/news/ftb-removal.md:11]: Haha ok I'll do it when I'm home

Comment (DioEgizio) [reply src/news/ftb-removal.md:11]: > Muggles don't know what merging a PR is. Say "with the next release" instead.

That feature isn't even developed yet, I think it's too early to say "with the next release"

**Review by txtsd — APPROVED** [review]
LGTM

File: src/news/ftb-removal.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+title: FTB Modpack Downloading Removal.
+description: Why FTB Modpack Downloading was Removed.
+date: 2023-04-27
+tags:
+  - News
+---
+
+At the request of Slowpoke, representing the FTB team, we have removed the feature that allows downloading FTB modpacks from Prism Launcher. [Github PR][pr-ftb-removal]
+
+Existing instances of FTB modpacks will remain functional, but with the next release, downloading FTB packs directly will no longer be possible.
```
Comment (Scrumplex) [inline src/news/ftb-removal.md:11]:
```suggestion
Existing instances of FTB modpacks will remain functional, but with the next release, downloading FTB packs within the launcher will no longer be possible.
```

File: src/news/ftb-removal.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
+---
+title: FTB Modpack Downloading Removal.
+description: Why FTB Modpack Downloading was Removed.
+date: 2023-04-27
+tags:
+  - News
+---
+
+At the request of Slowpoke, representing the FTB team, we have removed the feature that allows downloading FTB modpacks from Prism Launcher. [Github PR][pr-ftb-removal]
```
Comment (Scrumplex) [inline src/news/ftb-removal.md:9]:
```suggestion
At the request of Slowpoke, representing the FTB team, we have removed the feature that allows downloading FTB modpacks from Prism Launcher. [GitHub PR][pr-ftb-removal]
```

File: src/news/ftb-removal.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+Existing instances of FTB modpacks will remain functional, but with the next release, downloading FTB packs directly will no longer be possible.
+
+We will start working on a feature that allows importing FTB Modpacks from the FTB App, as this is acceptable to the FTB team.
```
Comment (Scrumplex) [inline src/news/ftb-removal.md:13]:
```suggestion
A way to import FTB Modpacks from the FTB App is planned, though no work was done yet. If you think you can help out, [join the development over here][issue-ftb-app-import]!
```

File: src/news/ftb-removal.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request6.png" style="width: 40em">
+<br/>
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request7.png" style="width: 40em">
+<br/>
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request8.png" style="width: 40em">
+
+~ PrismLauncher Team
+
+[pr-ftb-removal]: https://github.com/PrismLauncher/PrismLauncher/pull/1040
```
Comment (Scrumplex) [inline src/news/ftb-removal.md:45]:
```suggestion
[pr-ftb-removal]: https://github.com/PrismLauncher/PrismLauncher/pull/1040
[issue-ftb-app-import]: https://github.com/PrismLauncher/PrismLauncher/issues/235
```

File: src/news/ftb-removal.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+<br/>
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request5.png" style="width: 40em">
+
+### Additional
+
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request6.png" style="width: 40em">
+<br/>
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request7.png" style="width: 40em">
+<br/>
+<img alt="Screenshot of chat with FTB Team" src="/img/news/ftb-removal/request8.png" style="width: 40em">
```
Comment (Scrumplex) [inline src/news/ftb-removal.md:41]: The alt tags should probably be more descriptive, so people with vision impairments can let the screen reader read out the screenshots.

File: src/news/ftb-removal.md [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+
+~ PrismLauncher Team
```
Comment (Scrumplex) [inline src/news/ftb-removal.md:43]: We don't really do this in blog posts :D

**Review by Regular-Baf — APPROVED** [review]

File: src/news/ftb-removal.md (hunk 10 above)
Comment (TayouVR) [reply src/news/ftb-removal.md:226]: @IThundxr are you going to add this link too, or are you leaving it out intentionally?

**Review by DioEgizio — APPROVED** [review]

**Pull request merged**